src/renderer/pages/project/chat/graph.ts

```
@@ -47,6 +54,20 @@
             value: 0,
             update: ":counter.add(1)",
           },
+          messages: {
+            update: ":newMessages.array",
+          },
+          newMessages: {
+            agent: "pushAgent",
+            inputs: {
+              array: ":messages",
+              items: [{ role: "user", content: ":prompt"}, ":llm.message"],
```

ESLint

[error] Insert `·`

(prettier/prettier)